src/components/modals/AccelerateTxModal.tsx

```diff
@@ -45,118 +43,56 @@ export class AccelerateTxModalComponent extends PureComponent<Props, State> {
-        nativeAmount: spendTarget.nativeAmount,
-        publicAddress: spendTarget.publicAddress,
-        uniqueIdentifier: spendTarget.uniqueIdentifier
-      }))
-      const {
-        // Use the txid from the replaced transaction as the rbfTxid
-        txid: rbfTxid,
-        // Copy replaced transaction's metadata and swapInfo to the RBF transaction
-        metadata,
-        swapData
-      } = edgeTransaction
-
-      const edgeSpendInfo: EdgeSpendInfo = {
-        currencyCode,
-        spendTargets,
-        rbfTxid,
-        metadata,
-        swapData
-      }
+    let edgeSignedTransaction: EdgeTransaction = acceleratedTx
 
-      try {
-        const edgeUnsignedTransaction = await wallet.makeSpend(edgeSpendInfo)
+    this.setState({ status: 'sending' })
 
-        this.setState({
-          edgeUnsignedTransaction
-        })
-      } catch (error: any) {
-        this.setState({
-          error
-        })
-      }
-    } else {
-      const error = new Error('Missing spend target data.')
-      this.setState({
-        error
-      })
-    }
-  }
+    try {
+      // Sign, broadcast, and save the RBF transaction
+      edgeSignedTransaction = await wallet.signTx(acceleratedTx)
+      edgeSignedTransaction = await wallet.broadcastTx(edgeSignedTransaction)
+      await wallet.saveTx(edgeSignedTransaction)
 
-  signBroadcastAndSaveRbf = async () => {
-    const { wallet } = this.props
-    const { edgeUnsignedTransaction } = this.state
+      if (this.state.mounted) {
+        playSendSound().catch(error => console.log(error)) // Fail quietly
 
-    if (edgeUnsignedTransaction) {
-      let edgeSignedTransaction: EdgeTransaction = edgeUnsignedTransaction
+        this.setState({ status: 'sent' })
 
-      this.setState({ status: 'sending' })
+        showToast(s.strings.transaction_success_message)
 
-      try {
-        // Sign, broadcast, and save the RBF transaction
-        edgeSignedTransaction = await wallet.signTx(edgeUnsignedTransaction)
-        edgeSignedTransaction = await wallet.broadcastTx(edgeSignedTransaction)
-        await wallet.saveTx(edgeSignedTransaction)
-
-        if (this.state.mounted) {
-          playSendSound().catch(error => console.log(error)) // Fail quietly
-
-          this.setState({ status: 'sent' })
-
-          showToast(s.strings.transaction_success_message)
-
-          Actions.replace('transactionDetails', {
-            edgeTransaction: edgeSignedTransaction,
-            walletId: wallet.id
-          })
-        } else {
-          showWarning(s.strings.transaction_success_message)
-        }
-      } catch (error: any) {
-        console.log(error)
-
-        if (this.state.mounted) {
-          this.setState({ status: 'confirming' })
-          showError(error)
-        }
+        Actions.pop()
+        Actions.push('transactionDetails', {
+          edgeTransaction: edgeSignedTransaction,
+          walletId: wallet.id
+        })
```

Comment:
We are requiring folks to replace `Actions` with `navigation` when they edit components. So, please pass `navigation` as a prop and use that.